examples/azure-blob-storage/main.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from azure.storage.blob import BlobServiceClient
+from flask import Flask
+from flask_admin import Admin
+from flask_admin.contrib.fileadmin.azure import AzureFileAdmin
+from flask_babel import Babel
+from testcontainers.azurite import AzuriteContainer
+
+logging.basicConfig(level=logging.INFO)
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "secret"
+admin = Admin(app)
+babel = Babel(app)
+
+
+if __name__ == "__main__":
+    with AzuriteContainer() as azurite_container:
```

Comment:
Nice change - this feels helpful to get the example up and running quickly and reliably 👌